ibm/service/contextbasedrestrictions/resource_ibm_cbr_rule.go

```diff
@@ -35,7 +35,7 @@ func ResourceIBMCbrRule() *schema.Resource {
 			},
 			"contexts": &schema.Schema{
 				Type:        schema.TypeList,
-				Required:    true,
+				Optional:    true,
```

Comment:
is `contexts` still required, but it can be empty list? 

Here is `contexts` schema defined in cbr swagger's 
```
    NewRule:
      description: A rule
      type: object
      required:
        - contexts
        - resources
 ```